### PR TITLE
Patches

### PR DESCRIPTION
**Pull request by EPeillard**
Add a checkbox to limit the maximum of Influenced Set to 1.

### CODE REVIEW

File: EnhancePoE/View/MainWindow.xaml [hunk 1]
```diff
@@ -614,7 +614,18 @@
 									  Grid.Row="15"
 									  VerticalAlignment="Center"
 									  IsChecked="{Binding Source={x:Static properties:Settings.Default}, Path=hideOnClose, Mode=TwoWay}" />
-						</Grid>
+                            <TextBlock Grid.Column="1"
+									   Grid.Row="17"
+									   VerticalAlignment="Center"
+									   Text="Influenced Set Threshold:"
+									   ToolTipService.InitialShowDelay="50"
+									   ToolTip="Same as 'Full Set Threshold' but for Exalted Shard Receipe only. Leave at 0 for no threshold. " />
```
Comment (null77) [inline EnhancePoE/View/MainWindow.xaml]: type: Recipe

**Comment by HiMarioLopez** [comment]
Howdy, sorry for neglecting this PR for so long. What's your goal with this? Also may need to re-visit some of the influence stuff for 3.17 with new influence types coming in.

**Comment by EPeillard** [comment]
The goal was just to limit influence set to only one (to avoid having a bunch of body armors and no amulet). 
The changes might be outdated. I'll check that after the league start.

**Comment by HiMarioLopez** [comment]
Closing due to PR going stale. Appreciate your contributions, regardless

**Pull request closed without merging**